### PR TITLE
Snapshot Bar

### PR DESCRIPTION
**Pull request by ChaidGPT**
Added in the Snapshot Bar to Index and added it's styles to site.css

### CODE REVIEW

**Review by ParkerW98 — CHANGES_REQUESTED** [review]
This looks good! An issue that I'm seeing is the responsiveness of the bar itself, it seems to stay the same regardless of screen size. I would suggest using bootstrap to ensure that its responsive!

I am also slightly worried about the size of it and how it might work with the side bar. Ill be able to test this more once my design for it is done however.

**Comment by Chadloswald** [comment]
I'm always down to make things better/easier. 
So how do you use bootstrap? Can I convert what I've made to use that?

File: SuiteCareers/SuiteCareers/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -7,3 +7,52 @@
...
+<div id="snapshot-bar">                                     <!--The container for the whole dang thing-->
+
+    <div class="custom-date-range">                         <!--Container for the date search fields-->
+
+        <span>Filter by Date:</span>
+        <label for="from-date"></label>                     <!--What will be populated in the from-date field initially-->
+        <input type="date" id="from-date" name="from-date"> <!--Input field for "From Date"-->
+        <span>to</span>
+        <label for="to-date"></label>                       <!--What will be populated in the to-date field initially-->
+        <input type="date" id="to-date" name="to-date">     <!--input field for "To Date"-->
```
Comment (Bansenauer-Cascadia) [inline SuiteCareers/SuiteCareers/Views/Home/Index.cshtml]: Wrapped in a `<form asp-controller="Users" asp-action="filterByDate" method="post">...</form>` and include a "Filter" button

**Review by Bansenauer-Cascadia — CHANGES_REQUESTED** [review]
Good work Chad - I agree with Parker's note about responsive styling using bootstrap - note: the bootstrap library is already included in the base application

File: SuiteCareers/SuiteCareers/Views/Home/Index.cshtml (hunk 1 above)
Comment (ChaidGPT) [reply SuiteCareers/SuiteCareers/Views/Home/Index.cshtml]: If switching it to bootstrap makes it better than I can try. The problem is that I don't know how to do that

**Review by snholmes — CHANGES_REQUESTED** [review]
Looks good. I agree Bootstrap would be better. Since I've made the rest of the dashboard responsive, it would be nice if this piece was as well so the entire view is mobile-friendly.

**Review by darthcod3r — APPROVED** [review]
Looks awesome!

**Review by tylerjacobwilson — APPROVED** [review]

**Review by tylerjacobwilson — APPROVED** [review]

**Review by ParkerW98 — APPROVED** [review]
This looks great Chad! Excellent work on responsiveness and using Bootstrap. Fits wonderfully with the dashboard.

**Review by snholmes — APPROVED** [review]

**Review by Bansenauer-Cascadia — APPROVED** [review]
Good work

**Pull request merged**